file-formats/nont/type/zif_aff_nont_v1.intf.abap

```diff
@@ -0,0 +1,35 @@
+INTERFACE zif_aff_nont_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">General Information</p>
```

Comment:
I would put as title something containing "SAP Object Node Type" (or just skip it ;)). 

If you used "General Inforation" for UI purposes in ADT, you can overwrite it in you configuration file.